acceptorwrapper/base.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) TFG Co. All Rights Reserved.
```

Comment:
Uhm, we don't have the year in almost any file

Reply:
=/
I think every file should contain every year it had a modification, but I don't know if modifying the whole project is worth it.

Reply:
I used to do year of conception. Copyright can be tricky, it may be worth looking for proper advice.

If the entire project is like this, then I'd suggest leaving it as-is here and maybe adding the year in a separate PR.